TACC-Cloud/geoapi
# task/WG-286: use DesignSafe api for getting users

## Overview: ##

This PR now uses DesignSafe API for getting users from project information. The feature/tapisv2 branch was previously updated to handle the v3 response but we were using a fixture until we could test it. So this PR also removes the fixture and now uses DesignSafe's /api/projects/v2 to get project information on users.

## Related Jira tickets: ##

* [WG-286](https://tacc-main.atlassian.net/browse/WG-286)

## Testing Steps: ##
0. Use in conjunction with hazmapper (`feature/tapisv3`)
1. Create a map that is associated with a DesignSafe project
2.  Then in hazmapper, you can check that the map has the correct users associated with it. You could also check out the logs.
